File: Frontend/src/app/shared/models/event.ts
import { Address } from './address';
export class Event {
  id: number;
  name: string;
  description: string;
  eventDate: Date;
  eventAddress: Address;
  poster: string;

  constructor( id: number, name: string, description: string,
    eventDate: Date, eventAddress: Address, poster: string
  ){
    this.id = id;
    this.name = name;
    this.description = description;
    this.eventDate = eventDate;
    this.eventAddress = eventAddress;
    this.poster = poster;
  }
}

File: Backend/EventManager.Domain/Models/Event.cs
using System;

namespace EventManager.Domain.Models
{
    public class Event
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public DateTime EventDate { get; set; }
        public Address EventAddress { get; set; }
        public string Poster { get; set; }
    }
}
